trio_run_in_process/run_in_process.py

```diff
@@ -106,13 +75,10 @@ async def _monitor_state(
     if child_state is not State.FINISHED:
         raise InvalidState(f"Invalid final state: {proc.state}")
 
-    result = await coro_receive_pickled_value(from_child)
-
-    # The `returncode` should already be set but we do a quick wait to ensure
-    # that it will be set when we access it below.
-    with trio.fail_after(5):
-        await proc.wait_returncode()
+    logger.debug("Reading process result for %s", proc)
```

Comment:
It might be nice to use a common structure for all of these log messages so that they visually align.  I'm thinking:

```python
logger.debug("[%s] Reading result", proc)
logger.debug("[%s] Got return code:  return_code=%d", proc, return_code)
```

Reply:
It seemed like a good idea at first, but in practice it's highly unlikely these will come in immediate succession as there's an `await` in between them